.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ ubuntu-20.04, windows-latest, macOS-latest ]
+        os: [ ubuntu-latest, windows-latest, macOS-latest ]
```

Comment:
Why did you changed this?

It was on purpose testing an older version of Ubuntu with Python 3.8 and only the latest versions with the other Python versions.

Reply:
Ah ok, I thought it was a mistake as in [audeer](https://github.com/audeering/audeer/blob/master/.github/workflows/test.yml)  we use `ubuntu-latest` everywhere.

Reply:
Yes, it might be that we do not apply the same everywhere.

Reply:
I reverted the changes.